### PR TITLE
Fix seed regression, as option is not available in prev pytorch version

### PR DESCRIPTION
**Pull request by iseessel**
Our recommended pytorch version does not have seed available as an option for `torch.utils.data.distributed.DistributedSampler`. This was breaking master for me in distributed training on SLURM.

### CODE REVIEW

**Comment by prigoyal** [comment]
looks good to me.
